### PR TITLE
Turn on account confirmation for blazor server app template

### PR DESCRIPTION
**Pull request by HaoK**
Fixes https://github.com/aspnet/AspNetCore/issues/11886

cc @blowdart @danroth27 @javiercn @ajcvickers

### CODE REVIEW

**Comment by javiercn** [comment]
I think we also need to do this for the SPA templates too

**Comment by mkArtakMSFT** [comment]
@HaoK are you going to apply this to SPA templates too (per @javiercn 's comment above) ?

**Comment by HaoK** [comment]
Sure updated spa templates

**Review by javiercn — APPROVED** [review]

**Review by danroth27 — APPROVED** [review]
LGTM

File: src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs [hunk 1]
```diff
@@ -240,6 +240,20 @@ private void TestBasicNavigation(bool visitFetchData, bool usesAuth, IWebDriver
...
+
+                    // We will be redirected to the ConfirmEmail
+                    browser.Contains("/Identity/Account/ConfirmEmail", () => browser.Url);
+
+                    // Now we can login
+                    browser.FindElement(By.PartialLinkText("Login")).Click();
+                    browser.Exists(By.Name("Input.Email"));
+                    browser.FindElement(By.Name("Input.Email")).SendKeys(userName);
+                    browser.FindElement(By.Name("Input.Password")).SendKeys(password);
+                    browser.FindElement(By.Id("login-submit")).Click();
```
Comment (HaoK) [inline src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs]: @javiercn I must be doing something slightly wrong in updating the test, now that confirmation is required I had to add a few more clicks before the account should be logged in, but it doesn't look like the fetch data works still, anything obviously wrong with this test update?

Comment (javiercn) [reply src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs]: Can it be that the return url is getting lost in the flow and is not redirecting back to fetch-data?

I would recommend you try the flow manually with the latest preview9

Comment (HaoK) [reply src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs]: Was just missing a click on the fetch data at the end I guess, thanks

**Pull request merged**